GameServerCore/Domain/ISpell.cs

```diff
@@ -31,6 +31,7 @@ public interface ISpell: IUpdate
         void ApplyEffects(IAttackableUnit u, IProjectile p);
         void LevelUp();
         void AddProjectile(string nameMissile, float toX, float toY, bool isServerOnly = false);
```

Comment:
I suggest you to change one of the function to use the other one 'AddProjectile' will use your new 'AddProjectileCoords'.